projects/netgrif-components/src/lib/authentication/session-idle/session-idle.component.ts

```diff
@@ -0,0 +1,51 @@
+import {Component, Input} from '@angular/core';
+import {
+    AbstractSessionIdleComponent,
+    ConfigurationService,
+    LoggerService,
+    SessionIdleTimerService,
+    UserService
+} from '@netgrif/components-core';
+import {MatDialog} from "@angular/material/dialog";
+import {Router} from "@angular/router";
+import {PopupSessionIdleComponent} from "./popup-session-idle/popup-session-idle.component";
+
+@Component({
+    selector: 'nc-session-idle',
+    templateUrl: './session-idle.component.html',
```

Comment:
does this component have no content? Why is it a component then? Would a service not be better?